cdc/sink/dmlsink/mq/manager/kafka_manager.go

```diff
@@ -147,21 +139,17 @@ func (m *kafkaTopicManager) tryUpdatePartitionsAndLogging(topic string, partitio
 	}
 }
 
-func (m *kafkaTopicManager) getMetadataOfTopics(
+func (m *kafkaTopicManager) fetchNumPartitions(
```

Comment:
If I only look at the method name `fetchNumPartitions()` then I would expect a single number (maybe with an error) as return value. However here it is a map.

```suggestion
func (m *kafkaTopicManager) fetchNumPartitionsPerTopic(
```

```suggestion
func (m *kafkaTopicManager) fetchNumPartitionsByTopic(
```

This looks very similar to this, maybe use a similar name?
```go
func (a *saramaAdminClient) GetTopicsNumPartitions(
	_ context.Context, topics []string,
)
```


Feel free to ignore this if you think the name is fine as-is.

Reply:
This methods fetch number of partitions for all topics. 

I rename it to `fetchAllTopicsNumPartitions`